### PR TITLE
Add libselinux-python and libsemanage-python dependencies for nginx

### PR DESCRIPTION
**Pull request by humblearner**

### CODE REVIEW

File: roles/nginx/tasks/nginx_yum.yml [hunk 1]
```diff
@@ -1,3 +1,18 @@
+---
+- name: Dependency for nginx | libsemanage-python
```
Comment (Mierdin) [inline roles/nginx/tasks/nginx_yum.yml]: Just a little nit - why not put these into a single task using a list?

Comment (humblearner) [inline roles/nginx/tasks/nginx_yum.yml]: Sure, change on the way.

File: roles/nginx/tasks/nginx_yum.yml [hunk 2]
```diff
@@ -1,3 +1,11 @@
+---
+- name: Install dependencies for nginx
```
Comment (armab) [inline roles/nginx/tasks/nginx_yum.yml]: I believe it's not dependencies for nginx, but dependencies for http://docs.ansible.com/ansible/seboolean_module.html module to work.

See: https://github.com/StackStorm/ansible-st2/pull/92/files#diff-84ef76b67429d6f947d82255e8a3a5e5R36

Comment (armab) [inline roles/nginx/tasks/nginx_yum.yml]: Will adjust in https://github.com/StackStorm/ansible-st2/pull/93

**Pull request merged**